sawyer_description/urdf/sawyer_base.urdf.xacro

```diff
@@ -144,7 +170,9 @@
     <child link="right_l0"/>
     <axis xyz="0 0 1"/>
     <limit effort="80.0" lower="-3.0503" upper="3.0503" velocity="1.74"/>
-    <dynamics damping="5.0" friction="5.0"/>
+<xacro:if value="$(arg gazebo)">
+    <dynamics damping="15.0" friction="5.0"/>
```

Comment:
Is there a reason that damping is much higher on J0?

Reply:
Tuning magic... I agree these need to be revisited along with inertia and controller PID gains...